src/main/java/org/breedinginsight/brapi/v2/dao/BrAPIGermplasmDAO.java

```diff
@@ -112,9 +112,9 @@ public List<BrAPIGermplasm> getRawGermplasm(UUID programId) throws ApiException
         List<BrAPIGermplasm> cacheList = new ArrayList<>(programGermplasmCache.get(programId).values());
         return cacheList.stream().map(germplasm -> {
             germplasm.setGermplasmName(Utilities.appendProgramKey(germplasm.getDefaultDisplayName(), program.getKey(), germplasm.getAccessionNumber()));
-            if(germplasm.getAdditionalInfo() != null && germplasm.getAdditionalInfo().has(BrAPIAdditionalInfoFields.GERMPLASM_RAW_PEDIGREE)
-                    && !(germplasm.getAdditionalInfo().get(BrAPIAdditionalInfoFields.GERMPLASM_RAW_PEDIGREE).isJsonNull())) {
-                germplasm.setPedigree(germplasm.getAdditionalInfo().get(BrAPIAdditionalInfoFields.GERMPLASM_RAW_PEDIGREE).getAsString());
+            if(germplasm.getAdditionalInfo() != null && germplasm.getAdditionalInfo().has(GERMPLASM_RAW_PEDIGREE)
+                    && !(germplasm.getAdditionalInfo().get(GERMPLASM_RAW_PEDIGREE).isJsonNull())) {
+                germplasm.setPedigree(germplasm.getAdditionalInfo().get(GERMPLASM_RAW_PEDIGREE).getAsString());
```

Comment:
This is a personal preference, but I prefer to see `BrAPIAdditionalInfoFields.<constant>` as it tells me when I'm reading the code where the constant is located.  If I see a constant as you've changed it to, I'm expecting that constant to be defined in this class.

Reply:
No, but worth discussing as a group!